frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -275,7 +277,8 @@ export const EnvFromEditor = withDragDropContext(
           <EnvFromPairElement
             onChange={this._change}
             index={i}
-            nameString="config map/secret"
+            nameString={t('environment~Config Map/Secret')}
+            namePlaceholder={t('environment~config map/secret')}
```

Comment:
I might just skip the placeholder property and use `nameString` as-is without lowercasing. In this instance, we still want `ConfigMap/Secret`.

Reply:
Okay, I'll go through and remove it/update the integration tests.